canonical/cloud-init
# net: add the ability to blacklist network interfaces based on driver during enumeration of physical network devices

## Proposed Commit Message
net: add the ability to blacklist network interfaces based on driver during enumeration of physical network devices

## Additional Context
Relevant bug: https://bugs.launchpad.net/cloud-init/+bug/1874544

## Test Steps
Deploy a Standard_HB120rs_V2 instance on Azure
Install latest Mellanox OFED driver
Reboot and observe Traceback in log about ib0 and eth1 having duplicate mac addresses
Apply this PR's patch, reboot
Observe no traceback in cloud-init log

## Checklist:
